### ISSUE TITLE
ios Safari orientation change - section doesn't fill viewport when switching from landscape to portrait

### ISSUE THREAD

**Issue by mjwlist** [alvarotrigo/fullPage.js#3897]
### Description
I am using vue-fullpage. I have noticed that in ios safari when rotating from landscape to portrait the section comes up short in the viewport. I understand the issues presented by the safari top and bottom bars coming and going but having looked at your demos I see that some have the issue and others not. Please see below. 

Can you advise why link b works fine and link a has the resize issue. 

### Link to isolated reproduction
a) https://alvarotrigo.com/fullPage/#
b) https://alvarotrigo.com/fullPage/examples/navigationV.html

### Steps to reproduce it
Rotate link a to landscape and back to portrait. The section no longer fills the screen.

Rotate link b to landscape and back to portrait. The section does fill the screen.

### Versions 
iPhone 6s - ios 13.3.1 Safari

**Comment by BekzatJJ** [comment]
I have the same issue 

Any success on here?

**Comment by alvarotrigo** [comment]
Thanks for reporting it! 👍 
We will take a look at it for the next release!

**Comment by alvarotrigo** [comment]
I was able to reproduce the issue on the demo page (https://alvarotrigo.com/fullPage/) but not on https://alvarotrigo.com/fullPage/examples/navigationV.html.

I believe the issue you are talking about was related with the use of the option `responsiveHeight` that turns off the autoScrolling behaviour on landscape mode.
Then, when switching to portrait mode fullPage.js has to turn on the autoScrolling behaviour again and keep the active section. This is where it was failing. 

I've fixed it ;)
